sentry-android-ndk/src/main/jni/sentry.c

```diff
@@ -212,3 +220,123 @@ Java_io_sentry_android_ndk_SentryNdk_initSentryNative(JNIEnv *env, jclass cls,
 
     sentry_init(options);
 }
+
+JNIEXPORT void JNICALL
+Java_io_sentry_android_ndk_NativeModuleListLoader_nativeClearModuleList(JNIEnv *env, jclass cls) {
+    sentry_clear_modulecache();
+}
+
+JNIEXPORT jobjectArray JNICALL
+Java_io_sentry_android_ndk_NativeModuleListLoader_nativeLoadModuleList(JNIEnv *env, jclass cls) {
+    sentry_value_t image_list_t = sentry_get_modules_list();
```

Comment:
Maybe we drop the `_t`? or is this really a type

```suggestion
    sentry_value_t image_list = sentry_get_modules_list();
```

Reply:
well unless we rename `jobjectArray` but either of them needs a suffix or prefix, as they are the same thing but using a different data type